### ISSUE TITLE
Enhance Osiris report with further tabs on analysis page and pie charts

### ISSUE THREAD

**Issue by DavidRoy** [BiologicalRecordsCentre/Pantheon#15]
e.g. http://www.brc.ac.uk/pantheon/osiris/ecological-divisions?dynamic-sample_id=312083

Enhance by having ability to toggle between report format:
- Current boxes with numbers
- Pie chart

**Comment by johnvanbreda** [comment]
I would envisage writing a small "extension" to handle this so that the report data only loads once, but is rendered twice. We could put the boxes & pie chart versions on 2 tabs.

**Comment by HDean** [comment]
Notes from meeting 17/05/2016 - pie charts are not going to be used for Osiris reporting

**Comment by johnvanbreda** [comment]
Tabular format added - see example http://www.brc.ac.uk/pantheon/osiris/tabular?dynamic-sample_id=312083

**Comment by johnvanbreda** [comment]
Is the grouped conservation traits report basically the same output as the tabular format but laid out differently?

**Comment by johnvanbreda** [comment]
See Osiris and Isis report page.ppt
After teleconference we agreed the following. Hierarchical output:
Conservation status output referred to on page 5 (to add to the boxes in the hierarchy) is the number of species which have some sort of designation.
Some of the boxes in the hierarchy are really sub-headings, not actually traits. These will be coloured differently to indicate this.
Similarly some of the boxes relate to quality of the site and should be coloured differently.
The quality indicators and subheadings are detailed on Hannah's hierarchy.

Osiris tabular report output will now be 4 tabs:
1) Broad biotopes, 
2) Specific biotopes (with column linking each to BB).
3) Resources (with column linking each to BB).
4) SATs (with column linking each to BB).
We'll keep the existing "hierarchical tabular output" for now but it might not be needed.

Grouped conservation traits is the same information but filtered to only show traits from a certain group. This allows you to view Osiris information without it being cluttered with irrelevant indices, for example if a grassland species had fallen in the river you don't want a grassland quality index. You might pick the river group, and will then see the report limited to only the traits that are in this group. Jon to provide the list of traits which apply to 2 of these categories (rivers and wetlands).

**Comment by johnvanbreda** [comment]
I've added a tabbed report as described under the Tabular button. The SATs tab is currently missing as we've not done the new import yet so SATs aren't under the biotopes but that should be easy to add later. 

One issue - because the resources are sorted by descending species count, when there is a child resource (e.g. damp >> soil humidity) they will not always appear next to each other. I've included the parent term so things don't get too muddles.

Still to do: modifications to the Osiris overview panel output, grouped conservation traits, SATs tab.
